src/Framework/PaymentGateways/PaymentGateway.php

```diff
@@ -497,7 +270,11 @@ public function callRouteMethod($method, $queryParams)
      */
     private function isFunctionImplementedInGatewayClass(string $methodName): bool
     {
-        $reflector = new ReflectionMethod($this, $methodName);
+        try {
+            $reflector = new ReflectionMethod($this, $methodName);
+        } catch (ReflectionException $e) {
```

Comment:
Let's add a log here as this would be quite unexpected.

Reply:
resolved e237bd6e